src/reset_mol_ids.cpp

```diff
@@ -76,10 +77,14 @@ void ResetMolIDs::command(int narg, char **arg)
       offset = utils::tnumeric(FLERR,arg[iarg+1],1,lmp);
       if (offset < -1) error->all(FLERR,"Illegal reset_mol_ids command");
       iarg += 2;
+    } else if (strcmp(arg[iarg],"verbose") == 0) {
```

Comment:
please make this keyword "verbose yes/no" and instead of "verbose <integer>" and document it.